exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/Metadata.java

```diff
@@ -944,14 +940,16 @@ public ParquetTableMetadata_v2(ParquetTableMetadataBase parquetTable,
       this.directories = directories;
       this.columnTypeInfo = ((ParquetTableMetadata_v2) parquetTable).columnTypeInfo;
       this.drillVersion = DrillVersionInfo.getVersion();
-      this.isDateCorrect = true;
+      this.writerVersion = ParquetWriter.WRITER_VERSION;
     }
 
     public ParquetTableMetadata_v2(List<ParquetFileMetadata_v2> files, List<String> directories,
         ConcurrentHashMap<ColumnTypeMetadata_v2.Key, ColumnTypeMetadata_v2> columnTypeInfo) {
       this.files = files;
       this.directories = directories;
       this.columnTypeInfo = columnTypeInfo;
+      this.drillVersion = DrillVersionInfo.getVersion();
+      this.writerVersion = ParquetWriter.WRITER_VERSION;
```

Comment:
Given a set of files and directories, we're assuming the writer version? This is supposed to be the version of the Drill Parquet writer that wrote the actual Parquet files, right? Not the version of the metadata files that hold information about the Parquet files? Or, are we using the same writer version for both purposes? If we need two versions (one for Parquet, another for metadata), then we should introduce a separate Parquet metadata version. (Or, I'm misunderstanding the code...)